doc/tutorials/basic/index.md

```diff
@@ -46,15 +46,62 @@ Start your journey with these foundational tutorials:
 
 ## Master Panel Basics
 
-Once you're comfortable, it's time to dive deeper:
-
-| Part | Section A | Section B| Section C |
-|--------------------------|---------------------------|-------------------|------------------------------------------------------|
-| **1. Display Content**       | [`pn.panel`](pn_panel.md) | [Panes](panes.md) | [Performance Indicators](indicators_performance.md) |
-| **2. Organize Content**      | [Layouts](layouts.md)    | [Control the Size](size.md) | [Align Content](align.md)                           |
-| **3. Handle User Input**     | [Widgets](widgets.md)    | [React to User Input](pn_bind.md) | [Reactive Expressions](pn_rx.md)                        |
-| **4. Improve the look**      | [Templates](templates.md)| [Designs](design.md) | [Styles](style.md)                                |
-| **5. Improve the Feel**      | [Caching](caching.md)    | [Activity Indicators](indicators_activity.md) | [Progressive Updates](progressive_layouts.md)  |
+Once you're comfortable, it's time to dive deeper. Through a series of lessons we will learn about displaying content, arranging it on the page, handling user inputs and then how to improve the UI and UX of our applications.
```

Comment:
The purpose of the table was to make the page less "overwhelming" as that was the feedback we got. Now its back to a very long page. Is that on purpose?

Reply:
Yes, the table just made no sense to me. The columns were meaningless and it's a weird way to organize sections imo.